### PR TITLE
Removed Contributors.txt

### PR DESCRIPTION
**Pull request by dimitribouniol**
Updated contributors list.

### CODE REVIEW

**Comment by 0xTim** [comment]
I believe we can get rid of this - the rest of the Swiftlang repos are moving away from having a contributors list to just relying on things like GH's tools

**Comment by dimitribouniol** [comment]
Sounds like a plan!

**Comment by dimitribouniol** [comment]
@0xTim Actually, it seems like the license headers (both in this repo and NIO) still reference the Contributors.txt — should I update that to no longer reference one, or should we keep it in after all?

**Comment by 0xTim** [comment]
If we can remove it let's try that

**Review by 0xTim — APPROVED** [review]
One query otherwise LGTM

File: .mailmap
[the remark this answers is not in the document]
Comment (dimitribouniol) [reply .mailmap:1]: This one is actually used by git for authors UI — figured I'd leave it in?

[the remark this answers is not in the document]
Comment (0xTim) [reply .mailmap:1]: I was not aware! In that case, keep it

**Pull request merged**